### PR TITLE
Adding set -e on merge script

### PR DESCRIPTION
**Pull request by clebertsuconic**
I had some cases where I had pending changes on my local copy and the script silent failed.
This will make sure people using this script would notice any failure during a merge

### CODE REVIEW

File: scripts/merge-PR.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Setting the script to fail if anything goes wrong
+set -e
 
 # Use this to simplify the rebasing of PRs. PRs will be rebased during the merge on this process.
 # use: ./rebase-PR <PRID> textual description
```
Comment (johnament) [inline scripts/merge-PR.sh]: was `merge-PR.sh` moved from `rebase-PR`?

Comment (clebertsuconic) [inline scripts/merge-PR.sh]: what is rebase-PR?

Comment (clebertsuconic) [inline scripts/merge-PR.sh]: @johnament I wrote this to simplify how I was doing prs.. so others could do the same.

I'm curious now what is this rebase-PR that you mentioned.

Comment (johnament) [inline scripts/merge-PR.sh]: line 23 (now) of your script mentions `rebase-PR`.

Comment (clebertsuconic) [inline scripts/merge-PR.sh]: ahhhhh.. ok... 

It used to be called rebase-PR when it was on my ~home/bin only, before I sent as a commit :)

thanks for pointing it out.

Comment (clebertsuconic) [inline scripts/merge-PR.sh]: will fix it

**Pull request closed without merging**